tools/ci_build/github/linux/docker/Dockerfile.package_ubuntu_2004_gpu_ffmpeg

```diff
@@ -5,7 +5,7 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # Build base image with required system packages
-ARG BASEIMAGE=nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04
+ARG BASEIMAGE=nvidia/cuda:11.8.0-cudnn8-devel-ubuntu22.04
```

Comment:
Change the filename too (2004 to 2204).

Reply:
We are now using A10-12G as test machine, i don't know if this machine has updated to cuda12. if we can use cuda 12, then we don't need to build docker image for stable_diffusion job, because nvidia has docker image for cuda 12 and python3.10.

Reply:
It can.

Reply:
i tried cuda12 for stable diffusion job, it can't just change the docker image to cuda12, because the onnxruntime build with cuda11 and the built also related to llama and whisper job, so updating cuda version will involve many things to do.

So i think here we just keep the cuda 11 and maybe later updating to cuda12.